doc/index.rst

```diff
@@ -29,13 +29,13 @@ GHDL
    News
    ****
 
-   28.02.2022 - `GHDL v2.0 was released <https://github.com/ghdl/ghdl/milestone/10?closed=1>`__
+   2022.02.28 - `GHDL v2.0 was released <https://github.com/ghdl/ghdl/milestone/10?closed=1>`__
```

Comment:
Is reversing the date needed? The human readable format is day -> month -> year, except for the US, how does it completely unordered :).

Reply:
It's the international standard: https://en.wikipedia.org/wiki/ISO_8601. Incidentally, it's also the "natural" format in Basque. Nevertheless, I changed it because it makes sorting easier, both visually and programmatically (we already used two digits for all months).

Reply:
ok